bench/src/main/scala/cats/bench/TrampolineBench.scala

```diff
@@ -30,14 +31,12 @@ class TrampolineBench {
       y <- Trampoline.defer(trampolineFib(n - 2))
     } yield x + y
 
-  // TailRec[A] only has .flatMap in 2.11.
+   @Benchmark
```

Comment:
Good catch. Doing a search for "2.10" shows four or five other things like this we should do, but none of them look urgent.